projects/ad4630_fmc/Readme.md

```diff
@@ -1,8 +1,7 @@
-# AD4630-FMC HDL Project
+# AD4630-24 Eval FMC HDL Project
 
 Here are some pointers to help you:
-  * [Board Product Page](https://www.analog.com)
-  * Parts : [24-Bit, 2 MSPS, Dual Channel, Precision Differential SAR ADC](https://www.analog.com/ad4630-24)
-  * Project Doc: 
-  * HDL Doc: 
-  * Linux Drivers: https://wiki.analog.com/resources/tools-software/linux-drivers-all
+  * Parts: [24-Bit, 2 MSPS, Dual Channel, Precision Differential SAR ADC](https://www.analog.com/ad4630-24)
+  * Eval board user guide: https://wiki.analog.com/resources/eval/ad4630-24-eval-board
+  * HDL Doc: https://wiki.analog.com/resources/eval/user-guides/ad463x/hdl
+  * No-OS Drivers: https://wiki.analog.com/resources/tools-software/uc-drivers/ad463x
```

Comment:
The drivers part should be named Linux Drivers, even tough the driver is for No-OS, so that the current parser can get out the information.

Reply:
Done